generated/nidaqmx/constants.py

```diff
@@ -634,10 +634,10 @@ class ShuntCalSource(Enum):
 
 
 class ShuntElementLocation(Enum):
-    R_1 = 12465
-    R_2 = 12466
-    R_3 = 12467
-    R_4 = 14813
```

Comment:
Can you update CHANGELOG.md to mention this incompatibility. Don't worry too much about wordsmithing, we can take a final pass before release